homeassistant/components/zwave_js/api.py

```diff
@@ -571,6 +574,104 @@ def node_removed(event: dict) -> None:
     )
 
 
+@websocket_api.require_admin
+@websocket_api.websocket_command(
+    {
+        vol.Required(TYPE): "zwave_js/begin_healing_network",
+        vol.Required(ENTRY_ID): str,
+    }
+)
+@websocket_api.async_response
+@async_get_entry
+async def websocket_begin_healing_network(
+    hass: HomeAssistant,
+    connection: ActiveConnection,
+    msg: dict,
+    entry: ConfigEntry,
+    client: Client,
+) -> None:
+    """Begin healing the Z-Wave network."""
+    controller = client.driver.controller
+
+    @callback
+    def async_cleanup() -> None:
+        """Remove signal listeners."""
+        for unsub in unsubs:
+            unsub()
+
+    @callback
+    def forward_event(key: str, event: dict) -> None:
+        connection.send_message(
```

Comment:
Should we clean up when `key == "forward_event"`

Reply:
when would `key == "forward_event"`?

Reply:
oh my bad, I meant `result`.

Reply:
I guess it's fine, as long as someone is listening it should reeive events.